.github/workflows/validate_dockstore_yaml.yml

```diff
@@ -0,0 +1,38 @@
+name: validate-dockstore-yaml
+on: push
+jobs:
+    publish-test:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout branch
+              uses: actions/checkout@v3
+
+            - name: Setup Java 11
+              uses: actions/setup-java@v2
+              with:
+                distribution: 'temurin'
+                java-version: '11'
+
+            - name: Cache node_modules
+              uses: actions/cache@v2
+              with:
+                path: "**/node_modules"
+                key: ${{ runner.os }}-modules-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install dockstore cli
+              run: |
+                mkdir -p ~/bin
+                curl -L -o ~/bin/dockstore https://github.com/dockstore/dockstore-cli/releases/download/1.13.1/dockstore
+                chmod +x ~/bin/dockstore
+                export PATH=~/bin:$PATH
+                
+                mkdir -p ~/.dockstore
+                printf "token: 19dd4c1120940ea0f604cc856826d58eae108eeb72b8bb6186a8ce0e60b96637\nserver-url: https://dockstore.org/api\n" > ~/.dockstore/config
+                
+                dockstore --version
+
+            - name: Validate dockstore.yml
```

Comment:
I can try putting this in test/test_scripts/ as a tox test script instead of being its own git_action if needed